src/routes/graphql/schemas.ts

```diff
@@ -18,3 +20,347 @@ export const createGqlResponseSchema = {
+                const profiles = await prisma.profile.findMany();
+                return profiles;
+            },
+        },
+        profile: {
+            type: ProfileType,
+            args: {
+                id: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { id } = args;
+                const profile = await prisma.profile.findUnique({ where: { id: id }});
+                return profile || null;
+            },
+        },
+        getProfileByUserId: {
+            type: ProfileType,
+            args: {
+                userId: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { userId } = args;
+                const profile = prisma.profile.findUnique({ where: { userId: userId }});
+                return profile || null;
+            },
+        },
+        memberTypes: {
+            type: new GraphQLList(MemberType),
+            resolve: async (parent, args, { prisma }) => {
+                const memberTypes = await prisma.memberType.findMany();
+                return memberTypes;
+            },
+        },
+        memberType: {
+            type: MemberType,
+            args: {
+                id: { type: MemberTypeId },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { id } = args;
+                const memberType = await prisma.memberType.findUnique({ where: { id: id }});
+                return memberType || null;
+            },
+        },
+        /*prismaStats: {
+            type: PrismaStatsType,
+            resolve: async (fastify) => {
+                const prismaStats = await fastify.prismaStats
+                return { operationHistory: prismaStats } || null;
+            },
+        }, */
+        getPostByUserId: {
+            type: new GraphQLList(PostType),
+            args: {
+                userId: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { userId } = args;
+                const post = await prisma.post.findMany({ where: { authorId: userId }});
+                return post || null;
+            },
+        },
+        userSubscribedTo: {
+            type: new GraphQLList(UserType),
+            args: {
+                id: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma } ) => {
+                const { id } = args;
+                const subscriptions = await prisma.subscribersOnAuthors.findMany({ where: { subscriberId: id }});
+                if (subscriptions.length === 0) {
+                    return null;
+                }
+                const authorIds = subscriptions.map((subscription) => subscription.authorId);
+                return prisma.user.findMany({ where: { id: { in: authorIds}}});
+            }},
+        subscribedToUser: {
```

Comment:
Please take a look at `src/routes/users/_userId/subscribed-to-user/index.ts` how prisma call can be implemented in this case.